### ISSUE TITLE
Why does the keras tensorflow ranking example lack group_size variable?

### ISSUE THREAD

**Issue by mercarikaicheung** [tensorflow/ranking#275]
Hello,

Small question I was reviewing the documentation for the new keras api for tensorflow ranking and found there is some discrepancy between the estimator version.  The estimator version included the group_size variable, I was wondering why that has been left out of the Keras sample? 

Thanks guys for releasing this!!!!

**Comment by xuanhuiwang** [comment]
We now recommend to use the document interaction network model as shown in [this example](https://github.com/tensorflow/ranking/blob/master/tensorflow_ranking/examples/keras/antique_kpl_din.py#L296). The paper is [here](https://research.google/pubs/pub49364/) and it is based on MultiHeadAttention used in Transformer. The main advantage this model over the groupwise one is that the prediction is now deterministic

**Comment by mercarikaicheung** [comment]
> We now recommend to use the document interaction network model as shown in [this example](https://github.com/tensorflow/ranking/blob/master/tensorflow_ranking/examples/keras/antique_kpl_din.py#L296). The paper is [here](https://research.google/pubs/pub49364/) and it is based on MultiHeadAttention used in Transformer. The main advantage this model over the groupwise one is that the prediction is now deterministic

I see, yes there are threads about variance in inference mode, thanks for the quick response @xuanhuiwang!
